app/controllers/events_controller.rb

```diff
@@ -139,16 +139,12 @@ def index
 
       render json: EventSerializer.new(results, options).serialized_json, status: :ok
     else
-      aggregations = params.fetch(:aggregations, "") || ""
```

Comment:
is it truly necessary to stop supporting the `aggregation selection` functionality? This functionality doesn't have an impact in performance.

Reply:
The overall code for Elasticsearch aggregations is overly complex and I am not sure what value it adds. Right now the top priority is performance.